examples/freertos_linux/Linux_gcc_gcp_iot/README.md

```diff
@@ -6,22 +6,41 @@ This example uses the Google Cloud IoT Device SDK for Embedded C to connect a Fr
 
 Follow the steps below to connect the FreeRTOS application to the MQTT bridge.
 
-Before you begin, generate a [public/private key pair](https://cloud.google.com/iot/docs/how-tos/credentials/keys), store the private key in the `examples/freertos_linux/Linux_gcc_gcp_iot` directory, and name the key `ec_private.pem`. 
+1. Go to the repository's root directory.
 
-1. Run `make PRESET=FREERTOS_POSIX_REL` in the root directory of the repository. This command downloads of the [FreeRTOS kernel](https://www.freertos.org/index.html), downloads the [FreeRTOS Linux simulator](https://www.freertos.org/FreeRTOS-simulator-for-Linux.html), and ports the Device SDK to the FreeRTOS application.
+```
+cd iot-device-sdk-embedded-c
+```
+
+1. Generate a [public/private key pair](https://cloud.google.com/iot/docs/how-tos/credentials/keys), and store it in the example's directory.
+
+```
+openssl ecparam -genkey -name prime256v1 -noout -out examples/freertos_linux/Linux_gcc_gcp_iot/ec_private.pem
+openssl ec -in examples/freertos_linux/Linux_gcc_gcp_iot/ec_private.pem -pubout -out examples/freertos_linux/Linux_gcc_gcp_iot/ec_public.pem
```

Comment:
The instruction how to generate keys are on the page liked above, I wouldn't replicate it here again. This is just another possible point of failure.